inc/namespace.php

```diff
@@ -164,3 +165,25 @@ function lookup_file_sizes_as_needed_during_rest_response( $response, $post, $re
 
 	return $response;
 }
+
+/**
+ * Prints script to the head tag for passing PerformanceEntry
+ * objects to the postMessage method when adding an iframe.
+ *
+ * @return void
+ */
+function get_performance_entries() {
```

Comment:
Nitpick, we usually use `get_` in WP core for a function which returns things; since this outputs, I'd name it `render_`, `output_` etc.

Reply:
Good point, I updated the function name to `output_...`